### DISCUSSION TITLE
where can i find the SMAP_E_OPLsm soil moisture product added for Data Assimilation in LIS 7.5.0

### DISCUSSION THREAD

**Discussion by Devv21102001** [NASA-LIS/LISF#1616] [Q&A] [answered]
Hi, I'm trying to assimilate soil moisture and was looking at different datasets. I found that in LIS 7.5.0, SMAP_E_OPLsm was added to with the release notes decribing it as "Operation Enhanced (9-km) SMAP soil moisture (SMAP_E_OPL)".  When i tried finding a 9 Km SMAP dataset online I found  - https://nsidc.org/data/spl3smp_e/versions/4 . This however does not correspond to the number of latitude and longitude lines set in the SMAP_E_OPLsm code and the data format is also different with SMAP_E_OPLsm expecting NetCDF and the data I found being in HDF5. The dataset I found on the website can however be assimilated using the code in the NASA_SMAPsm directory. Could someone please tell me which dataset SMAP_E_OPLsm corresponds to and help me locate it online? Thanks!

**Comment by dmocko** [comment, marked as answer]
Hi @Devv21102001 

The SMAP_E_OPLsm soil moisture is not publicly available, as the data is only available for one of our projects.

We recommend that you use the SMAP_NRTsm data, which is available here:

https://daacdata.apps.nsidc.org/pub/DATASETS/SMAP/SPL2SMP_NRT.107/

**Reply by Devv21102001** [reply]
Alright, Thanks a lot!
